<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
	background-image: url(:/resrcs/imgs/gallery/blue-designed-grunge-concrete-texture-vintage-background-with-space-text-image.jpg)
}</string>
  </property>
  <widget class="QLabel" name="label_pic1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20%;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
